CHANGELOG

```diff
@@ -1,3 +1,6 @@
+0.3.3
+ - enh: add first GUI that supports transfer of RTDC data (#3)
```

Comment:
The GUI is a feature ("feat:"), please increment the minor version and set the patch to 0 (i.e. 0.4.0).